gdk4/Gir.toml

```diff
@@ -449,8 +449,19 @@ status = "generate"
         name = "region"
         const = true
     [[object.function]]
+    pattern = "set_(color_state|display|fd|fourcc|height|modifier|n_planes|offset|premultiplied|stride|update_region|update_texture|width)"
+    # Take by value and return self
+    manual = true
+    [[object.function]]
+    pattern = "get_fd"
+    # Use BorrowedFd
+    manual = true
+    [[object.function]]
     name = "build"
     manual = true # Can't be auto-generated
+    [[object.property]]
+    pattern = "(color-state|display|fourcc|height|modifier|n-planes|premultiplied|update-region|update-texture|width)"
```

Comment:
There are certain props here i would probably keep their getters as it can be quite useful for some cases. Eg, https://gitlab.gnome.org/GNOME/libmks/-/blob/main/lib/mks-dmabuf-paintable.c#L194

Reply:
The properties were just used for the `notify` signals. `update-region` (and everything else) has proper getters that are still there

Reply:
@bilelmoussaoui This is good to go then or do you see a problem? :)

Reply:
Looks like the Windows build is generally broken. Shouldn't really try building sysprof...